platform/android/detect.py

```diff
@@ -205,7 +205,7 @@ def mySpawn(sh, escape, cmd, args, env):
 
     env.Append(CPPFLAGS=["-isystem", env["ANDROID_NDK_ROOT"] + "/sources/cxx-stl/llvm-libc++/include"])
     env.Append(CPPFLAGS=["-isystem", env["ANDROID_NDK_ROOT"] + "/sources/cxx-stl/llvm-libc++abi/include"])
-    env.Append(CXXFLAGS=["-std=gnu++14"])
+    env.Append(CXXFLAGS=["-std=c++17"])
```

Comment:
We set `gnu++17` in `SConstruct` already, is there any reason to override it and change to `c++17`?

Reply:
No, I just picked `c++17` by mistake. I'm going to remove this line.